connector/connect/common/src/main/protobuf/spark/connect/ml.proto

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = 'proto3';
+
+package spark.connect;
+
+import "spark/connect/expressions.proto";
+import "spark/connect/relations.proto";
+import "spark/connect/ml_common.proto";
+
+option java_multiple_files = true;
+option java_package = "org.apache.spark.connect.proto";
+
+// a MlCommand is a type container that has exactly one ML command set
+message MlCommand {
+  oneof ml_command_type {
+    // call `estimator.fit` and returns a model
+    Fit fit = 1;
+    FetchModelAttr fetch_model_attr = 2;
+  }
+
+  message Fit {
+    MlStage estimator = 1;
+    Relation dataset = 2;
+  }
+
+  message FetchModelAttr {
+    ModelRef model_ref = 1;
+    string name = 2;
+  }
+}
+
+message MlCommandResponse {
+  oneof ml_command_response_type {
+    Expression.Literal literal = 1;
+    ModelInfo model_info = 2;
+    Vector vector = 3;
+    Matrix matrix = 4;
+    MlStage stage = 5;
+    ModelRef model_ref = 6;
+    bool is_dataframe = 7;
+  }
+  message ModelInfo {
+    ModelRef model_ref = 1;
+    string model_uid = 2;
```

Comment:
What's the difference b/w model_ref and model_uid?

Reply:
removed the model_uid